src/Microsoft.HttpRepl/HttpState.cs

```diff
@@ -117,7 +135,7 @@ public Uri GetEffectivePathForPrompt()
                 return null;
             }
 
-            return GetEffectivePath(BaseAddress, string.Join('/', PathSections.Reverse()), "");
+            return GetEffectivePath(BaseAddress, string.Join('/', PathSections.Reverse()), "", queryParam: null);
```

Comment:
I wonder about whether we should display the query params for the prompt or not (currently it does not). But if it is not displayed on the prompt, then you can't really see what the actual URL is that will be executed if you do a GET or POST. 

Perhaps we could defer that and have a separate PR that made it an option for whether or not to display query params.

Reply:
I like the thought of separating these concerns and creating an issue with it and I'll get to implementing a sort of optional echo feature.